listener/src/event_handler/deposit_handler.rs

```diff
@@ -0,0 +1,44 @@
+use failure::Error;
+use slog::Logger;
+use std::sync::Arc;
+
+use dfusion_core::models::PendingFlux;
+
+use graph::components::ethereum::EthereumBlock;
+use graph::components::store::EntityOperation;
+use graph::data::store::{Entity};
+
+use web3::types::{Log, Transaction};
+
+use super::EventHandler;
+use super::util;
+
+#[derive(Debug, Clone)]
+pub struct DepositHandler {}
+
+impl EventHandler for DepositHandler {
+    fn process_event(
+        &self,
+        logger: Logger,
+        _block: Arc<EthereumBlock>,
+        _transaction: Arc<Transaction>,
+        log: Arc<Log>
+    ) -> Result<Vec<EntityOperation>, Error> {
+        let entity_id = util::entity_id_from_log(&log);
+        let flux = PendingFlux::from(log);
+        
+        info!(logger, "Processing Deposit {:?}", &flux);
```

Comment:
This is a cool log. Just a question here, is there another (more generic) log statement like 

```
"{} - event received" % event.name
```

Reply:
Unfortunately we only know the keccak hash of the log (not the reverse plain text name). Therefore I'm expecting each handler to know which plaintext event they are listening to.